tests/cluster/tests/includes/init-tests.tcl

```diff
@@ -44,6 +44,7 @@ test "Cluster nodes hard reset" {
         R $id config set repl-diskless-load disabled
         R $id config set cluster-announce-hostname ""
         R $id DEBUG DROP-CLUSTER-PACKET-FILTER -1
+        R $id DEBUG CLOSE-CLUSTER-LINK-ON-PACKET-DROP 0
```

Comment:
```suggestion
```

I assume this is no longer needed now that the tests were moved?

Reply:
Yes, not strictly needed. Will remove it.